shared/fs/common/static-breadcrumb.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import * as React from 'react'
+import {globalStyles, globalColors, globalMargins} from '../../styles'
+import {Box, Icon, Text, type IconType} from '../../common-adapters'
+
+const getIcon = (tlfType: string): IconType => {
+  switch (tlfType) {
+    case 'private':
+      return 'icon-folder-private-16'
+    case 'public':
+      return 'icon-folder-public-16'
+    case 'team':
+      return 'icon-folder-team-16'
+    default:
+      return 'iconfont-question-mark'
+  }
+}
+
+type Props = {
+  pathElements: Array<string>,
+  showTlfTypeIcon?: boolean,
+  includeLast?: boolean,
+}
+
+const StaticBreadcrumb = ({pathElements, showTlfTypeIcon, includeLast}: Props) => (
+  <Box style={stylesBox}>
+    {[
+      showTlfTypeIcon && (
+        <Icon
+          type={getIcon(pathElements[1])}
+          color={globalColors.blue}
+          style={stylesIconFolderType}
+          key="icon"
+        />
+      ),
+      <Text key="text" type="BodySmallSemibold">
+        {pathElements[1]}
+      </Text>,
+      ...pathElements.slice(2, includeLast ? undefined : pathElements.length - 1).map((elem, idx) => [
```

Comment:
What happens if `pathElements` has length 2 and `includeLast` is false?

Reply:
It'd be an empty slice, so should be fine:

```
%  node
> [1,2].slice(2, 1)
[]
```